podium-gateway/src/main/java/nl/thehyve/podium/web/rest/RequestResource.java

```diff
@@ -176,6 +178,27 @@
     }
 
     /**
+     * Fetch the request
+     *
+     * FIXME: Add new annotation that check whether an member (coordinator or reviewer) of an organisation has access
+     *        to a different resource such as a Request.
+     *
+     * @param uuid of the request
+     * @throws URISyntaxException Thrown in case of a malformed URI syntax
+     * @throws ActionNotAllowedInStatus when a requested action is not available for the status of the Request
+     * @return The list of requestDTOs generated
+     */
+    @SecuredByCurrentUser
```

Comment:
This doesn't make sense, see https://github.com/thehyve/podium/blob/dev/podium-common/src/main/java/nl/thehyve/podium/common/security/annotations/SecuredByCurrentUser.java.
That annotation expects there to be a user uuid parameter or object that is associated with a user (could be a request). Does not apply to request uuid.